### ISSUE TITLE
[Fixed in GH-6052] Hidden Visual Focus on Section4 banners

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4297]
![Screen Shot 2019-08-28 at 10 28 01 PM](https://user-images.githubusercontent.com/15113219/63905818-69206280-c9e3-11e9-9654-eaadf98c3f32.png)

Element receives visual focus but the image hides it partially

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 41 59 PM](https://user-images.githubusercontent.com/15113219/73576977-bfd61080-4449-11ea-958b-38eb94ddca12.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
